<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QDialog" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>273</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>168</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="registrationpushButton">
      <property name="text">
       <string>Registration</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="returnpushButton">
      <property name="text">
       <string>Return</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>441</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="accountlabel">
        <property name="text">
         <string>Accout：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="accoutnamelabel">
        <property name="text">
         <string>AccoutName：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="passwordlabel">
        <property name="text">
         <string>Password：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="confirmpasswordlabel">
        <property name="text">
         <string>ConfirmPass：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="verificationlabel">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 10, 42);

font: 75 24pt &quot;Aharoni&quot;;</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="accoutlineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="accoutnamelineEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passwordlineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="confirmpasslineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="verificationlineEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>212</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QRadioButton" name="warehouseradioButton">
      <property name="text">
       <string>Warehouse</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="inventoryradioButton">
      <property name="text">
       <string>Inventory</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>inventoryradioButton</zorder>
  <zorder>warehouseradioButton</zorder>
  <zorder>horizontalSpacer_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
